haystack/modeling/model/feature_extraction.py

```diff
@@ -40,12 +42,16 @@
 SPECIAL_TOKENIZER_CHARS = r"^(##|Ġ|▁)"
 
 
+if not is_imported("transformers"):
+    TOKENIZER_MAPPING_NAMES = {}
+    FEATURE_EXTRACTOR_MAPPING_NAMES = {}
```

Comment:
Why do this? Shouldn't we also add a `try/catch` when importing `transformers` for this to work?